plugin.json

```
@@ -31,6 +31,64 @@
             "help_text": "The user to use as part of the demo plugin, created automatically if it does not exist.",
             "placeholder": "demo_plugin",
             "default": "demo_plugin"
+        }, {
+            "key": "LastName",
+            "display_name": "Demo User Last Name",
+            "type": "radio",
+            "help_text": "Select the last name for the demo user",
+            "default": "Plugin User",
+            "options": [{
+                "display_name": "Plugin User",
+                "value": "Plugin User"
+            }, {
+                "display_name": "Demoson III",
+                "value": "Demoson III"
+            }, {
+                "display_name": "McDemo",
+                "value": "McDemo"               
+            }]
+        }, {
+            "key": "TextStyle",
+            "display_name": "Text Style",
+            "type": "dropdown",
+            "help_text": "Change the text style of the messages posted by this plugin",
+            "default": "none",
```

The default value reference a value, not the display name.
 
```suggestion
            "default": "",
```

---

Oh, good catch. Sorry, that didn't manifest as an issue for me because I had changed the setting multiple times and kept redeploying the plugin.

This has been corrected.